cmake/modules/FindIso9660pp.cmake

```diff
@@ -12,31 +12,47 @@
 
 if(PKG_CONFIG_FOUND)
   pkg_check_modules(PC_ISO9660PP libiso9660++>=2.1.0 QUIET)
+  pkg_check_modules(PC_CDIOPP libcdio++>=2.1.0 QUIET)
   pkg_check_modules(PC_ISO9660 libiso9660>=2.1.0 QUIET)
 endif()
 
+find_package(Cdio)
+
 find_path(ISO9660PP_INCLUDE_DIR NAMES cdio++/iso9660.hpp
-                                PATHS ${PC_ISO9660PP_INCLUDEDIR})
+                                PATHS ${PC_ISO9660PP_INCLUDEDIR} ${CDIO_INCLUDE_DIR})
 
 find_library(ISO9660PP_LIBRARY NAMES libiso9660++ iso9660++
                                PATHS ${PC_ISO9660PP_LIBDIR})
 
+find_path(CDIOPP_INCLUDE_DIR NAMES cdio++/cdio.hpp
```

Comment:
I don't really understand why you want to check for a header that we don't explicitly need?

Reply:
It's the first `#include` in `libiso9660++` header, i.e. it's used indirectly by `libiso9660++`.

Reply:
I really think that is something that distros have to take care of. I.e., not our problem.

Reply:
(AFAIK we don't do this for other dependencies, as well)

Reply:
Yep, proper dependencies in distro packages should works fine. Actually it could be removed.

Reply:
Initially it was added because I had added library binary file as well, but later it was discovered that binary is not required.